sentry-android-integration-tests/test-app-sentry/build.gradle

```
@@ -0,0 +1,48 @@
+plugins {
+    id 'com.android.application'
+}
+
+android {
+    compileSdk 32
+
+    defaultConfig {
+        applicationId "io.sentry.java.tests.perf.appsentry"
+        minSdk 21
+        targetSdk 32
+        versionCode 1
+        versionName "1.0"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled true
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            signingConfig signingConfigs.debug
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    buildFeatures {
+        viewBinding true
+    }
+    signingConfigs {
+        debug {
+            storeFile file("debug.keystore")
+            storePassword = "android"
+            keyAlias = "androiddebugkey"
+            keyPassword = "android"
+        }
+    }
```

This should fix the debug error for Macrobenchmark (the same should be applied to the other module, too)
```suggestion
    }
    
    variantFilter {
        if (Config.Android.shouldSkipDebugVariant(buildType.name)) {
            ignore = true
        }
    }
```

---

Do you know whether there's any real benefit going with Macrobenchmark, especially since the other approach (Appium) is already implemented. As in, for example, does it provide more precise results? So that we don't throw out work already done just for the sake of using builtin tooling, if it doesn't give us anything. On the other hand, we may lose some reusability, considering the Appium project can also be used to test iOS, RN, Flutter apps, with some adjustments.